config/optional/workflows.workflow.editorial.yml

```diff
@@ -32,6 +32,8 @@ type_settings:
     archive:
       label: Archive
       from:
+        - draft
```

Comment:
@vincent-gao is it a requirement that a Draft can be archived?

Reply:
yes @sonnykt . it is